migrations/20230626154913_edit_product.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE PROCEDURE `EditProduct`(
+    IN inProductId int,
+    IN inProductName VARCHAR(50),
+    IN inProductDescription VARCHAR(255),
+    IN inProductPrice FLOAT
+)
+BEGIN
+
+    IF (SELECT id FROM products WHERE id = inProductId) IS NULL THEN
+        SELECT 0;
```

Comment:
you can put everything in one IF

IF EXISTS(..) THEN

SELECT 1
END IF;
In code if select from db is != 1 then it is not updated